compiler-rt/test/profile/instrprof-gc-sections.c

```diff
@@ -1,7 +1,7 @@
 // REQUIRES: linux, lld-available
 
 // FIXME: Investigate and fix.
-// XFAIL: powerpc64-target-arch
+// XFAIL: powerpc64-target-arch,powerpc64le-target-arch
```

Comment:
This is only an issue on the 64bit arch? What about 32bit?

Reply:
It affects anyone that had lld in their environment but but does not want to test it, this is only fixes the effected bots.